ready-to-use fields/README.md

```diff
@@ -0,0 +1,15 @@
+<img align="right" src="https://cdn.miranj.in/mc3/img/apple-touch-icon-precomposed.png" width="57" height="57" alt="Miranj">
+
+# Miranj Craft Boilerplate
+
+A bunch of ready to use fields provided by Miranj to speed-up your project development process. It uses the [Field Manager] plugin (https://github.com/verbb/field-manager) to import the fields.
+
+## Usage
+
+1.  Open the Control Panel and go to _Field Manager_ (see step*1 in \_images* folder).
```

Comment:
Are the instructions to import fields not documented in the plugin’s docs somewhere? Can we not link to that directly instead of repeating the steps here?

Reply:
There is [documentation](https://verbb.io/craft-plugins/field-manager/docs/feature-tour/field-manager#import) for import fields but I think it's very little. So I added steps.